src/data-access/database.js

```diff
@@ -0,0 +1,14 @@
+const Sequelize = require('sequelize');
+
+module.exports =  new Sequelize('users', 'postgres', 'Marko2021', {
```

Comment:
@lenon777, login, password, host, and other connection details have come from environment variables, furthermore, it is a security leak storing connection details to the database in source code.